cartography/intel/aws/lambda_function.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
```

Comment:
We should rename this file to lambda.py since that's how AWS calls it.

Reply:
I tried lambda.py but the name lambda would collide with the existing python lambda function. I think lambda is a reserved keyword

Reply:
Oh lol, lambda_function.py works then